### DISCUSSION TITLE
How to change vertical alignment of portraits

### DISCUSSION THREAD

**Discussion by bvoightlab** [greenelab/lab-website-template#161] [Q&A] [answered]
I'm setting up a listing of alumni on a teams page: voightlab.com/team/#alumni

and I can't seem to see how to get the spacing to be uniform in a row so that all of the images are lined up

e.g., like https://greenelab.com/team/#alumni

i.e. somehow gallery in the greenelab website knows that the row will have three 'text rows' for row 1, so will align all images to the "top"

whereas mine doesn't want to do that by default :/

very minor announce in an otherwise very helpful website template

-BV

**Comment by vincerubinetti** [comment]
I assume this is the website and page that you're talking about:

https://voightlab.com/team/

If I'm understanding what you're saying, you want the tops of each portrait to be aligned when the name text may be any number of rows. To do this you can simply remove the `vertical-align: middle` style from your `.portrait` class in `portrait.scss`, or you can explicitly set it to `top`. 

I haven't updated greenelab.com in a while, and I guess I added `vertical-align: middle` to the template at a later point because I thought it looked better, but now I think top looks better. Actually I just did a major update to the template that you may want to upgrade to and it also removed the middle align.

**Comment by bvoightlab** [comment, marked as answer]
This did the trick! Thanks v much.
once I get my content populated, will take a look at updating the template
